nixos/modules/services/networking/ndppd.nix

```diff
@@ -91,7 +100,7 @@ let
           auto: Same as iface, but instead of manually specifying the outgoing
             interface, check for a matching route in /proc/net/ipv6_route.
         '';
-        default = "auto";
+        default = "static";
```

Comment:
I think this should be documented in the 25.05 release notes.

Reply:
Done.